pages/common/jpegoptim.md

```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
+
+- Optimise a set of jpeg images:
+
+`jpegoptim {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Optimise jpeg images, stripping all non-essential information:
+
+`jpegoptim --strip-all {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be progressive:
+
+`jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
```

Comment:
Why is there an "all" in all-progressive? Is there an option where not all of the images are made progressive?